osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json

```diff
@@ -40,6 +40,130 @@
         }
       ]
     },
+    {
+      "tag": "ACTIVE_ENERGY_IMPORT_RATE_1",
+      "description": "Active energy import (+A) rate 1",
+      "note": null,
+      "class-id": 3,
+      "version": 0,
+      "obis": "1.0.1.8.1.255",
+      "group": "ELECTRICITY",
+      "meterTypes": [
+        "SP",
+        "PP"
+      ],
+      "attributes": [
+        {
+          "id": 2,
+          "description": "value",
+          "datatype": "double-long-unsigned",
+          "valuetype": "DYNAMIC",
+          "value": "0",
+          "access": "R"
+        },
+        {
+          "id": 3,
+          "description": "scaler_unit",
+          "datatype": "scal_unit_type",
+          "valuetype": "FIXED_IN_PROFILE",
+          "value": "0, WH",
+          "access": "R"
+        }
+      ]
+    },
+    {
+      "tag": "ACTIVE_ENERGY_IMPORT_RATE_2",
+      "description": "Active energy import (+A) rate 2",
+      "note": null,
+      "class-id": 3,
+      "version": 0,
+      "obis": "1.0.1.8.1.255",
+      "group": "ELECTRICITY",
+      "meterTypes": [
+        "SP",
+        "PP"
+      ],
+      "attributes": [
+        {
+          "id": 2,
+          "description": "value",
+          "datatype": "double-long-unsigned",
+          "valuetype": "DYNAMIC",
+          "value": "0",
```

Comment:
value staat niet op 0 in de spec, maar "Rate 2 = normal/high tariff" moeten we daar wat mee?

Reply:
The value of this register holds the meter value for rate 1, not the value "Rate 2 = normal/high tariff". This is unclear in the P3 document. Since the meter value is dynamic (it will be updated by te meter), the value need to be removed.